renv/core.py

```diff
@@ -132,16 +131,33 @@ def ensure_directories(self, env_dir):
         r_script = "Rscript"
         context.R_exe = r_exe
         context.R_script = r_script
-        context.R_version = os.path.split(self.r_path)[1]
         if self.r_bin_path:
             context.abs_R_exe = os.path.join(self.r_bin_path, r_exe)
             context.abs_R_script = os.path.join(self.r_bin_path, r_script)
         else:
             context.abs_R_exe = os.path.join(self.r_path, "bin", r_exe)
             context.abs_R_script = os.path.join(self.r_path, "bin", r_script)
         context.abs_R_path = self.r_path
-        logging.info(f"System R(version):  {self.r_path}({context.R_version})")
 
+        # Get the version of R
+        # Major Version
+        Rcmd = f"{context.abs_R_script} " \
+               f"-e \'R.version$major\'"
+        recommended_pkgs = subprocess.Popen([Rcmd], stderr=subprocess.PIPE, stdout=subprocess.PIPE,
+                                            shell=True, encoding='utf-8')
+        error = recommended_pkgs.stderr.readlines()
```

Comment:
stdout, stderr = p.communicate() ?

Reply:
Worked on this in the latest commit, but did not test.